dispatchr-ruby/app/controllers/requests_controller.rb

```diff
@@ -31,6 +28,8 @@ def create
 		@request.user_id = request_params[:user_id]
 		@request.expiration_date = request_params[:expiration_date]
 		@request.request_items = request_items
+		@request.hero_id = request_params[:hero_id]
```

Comment:
We need a whole new endpoint for this. Maybe /requests/:id. When a user creates a request we shouldn't need to add the hero_id and status should just default to open probably. On this new endpoint is where you'll add the hero_id and change the status